src/GraphQl/Resolver/Factory/ItemMutationResolverFactory.php

```diff
@@ -95,20 +103,37 @@ public function __invoke(string $resourceClass = null, string $rootClass = null,
                     $context += $resourceMetadata->getGraphqlAttribute($operationName, 'denormalization_context', [], true);
                     $item = $this->normalizer->denormalize($args['input'], $resourceClass, ItemNormalizer::FORMAT, $context);
                     $this->validate($item, $info, $resourceMetadata, $operationName);
+
+                    if (null !== $this->dispatcher) {
+                        $this->dispatcher->dispatch(PreWriteEvent::NAME, new PreWriteEvent($operationName, $item));
```

Comment:
Guess there will be an issue here, since the `$operationName` from GraphQL (`create`, `update`, `delete`) is not the same as the HTTP methods used in REST